Assets/Scripts/Editor/BuildScript.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Editor
+{
+    public class BuildScript
+    {
+        static void Windows()
+        {
+            
+            string buildPath = Environment.GetEnvironmentVariable("BUILD_PATH");
+            if (string.IsNullOrEmpty(buildPath))
+                buildPath = @"C:\bob\BoB.exe";
+            
```

Comment:
Path as a parameter instead of const variable?

Reply:
it is as a parameter, but if its not set it will take this value